pubspec.yaml

```diff
@@ -1,7 +1,13 @@
 name: countly_flutter
 description: Countly is an innovative, real-time, open source mobile analytics and push notifications platform.
 version: 23.8.4
-homepage: https://support.count.ly/hc/en-us/articles/360037944212-Flutter
+homepage: https://support.count.ly/hc/en-us/articles/360037944212
+repository: https://github.com/Countly/countly-sdk-flutter-bridge
+issue_tracker: https://github.com/Countly/countly-sdk-flutter-bridge/issues
+documentation: https://support.count.ly/hc/en-us/articles/360037944212
+platforms:
```

Comment:
Hi peter! it this platforms part necessary?

Reply:
We can omit it but then we're allowing pub.dev to determine the platforms our sdk is available on automatically.

Reply:
should I remove it?

Reply:
Awesome, if it is useful it should stay 👍